content/geometry/onSegment.h

```diff
@@ -4,6 +4,7 @@
  * License: CC0
  * Source: https://vlecomte.github.io/cp-geo.pdf
  * Description: Returns true iff p lies on the line segment from s to e. Intended for use with e.g. Point<ll> where overflow is an issue.  Use \texttt{(segDist(s,e,p)<=epsilon)} instead when using Point<double>.
+ * Works with integers.
```

Comment:
There's already a comment about about integers/doubles; I don't think this adds anything. (That comment could be reformulated to drop the "e.g." and "where overflow is an issue"; `Point<int>` should die)

Reply:
Dropping those parts results in `Intended for use with Point<ll>`. I think `works with integers` is a clearer way of expressing the same thing.

Reply:
I removed even the "Works with integers." -- I think it's pretty implicit from how most geometry algorithms operate on ints, and how the description says how to modify it for doubles.